doc/environment.yml

```diff
@@ -13,3 +13,4 @@ dependencies:
   - pip:
     - pydata-sphinx-theme
     - sphinx_favicon
+    - sphinx_copybutton
```

Comment:
why these packages are not installed with conda, but with pip?
I installed them using conda and it works.

See hydroroot